### ISSUE TITLE
Incorrect role(menu) defined for the controls in "Command bar" screen.

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#12282]
User Experience:
Keyboard users will face difficulty as screen reader is announcing incorrect role for controls in command bar screen.

Repro Steps:
Step 1: Open https://fabricweb.z5.web.core.windows.net/oufr/7.101.0/#/examples/commandbar in edge browser
Step 2: Navigate to buttons in the screen using arrow keys and verify whether the screen reader is announcing the correct role or not.

Actual:
Screen reader is announcing incorrect role for buttons in the screen.
Screen reader is announcing as "Menu item".

Expected.
Screen reader should announce as "Upload button".

[menu item control.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/4324443/menu.item.control.zip)

**Comment by aneeshack4** [comment]
This is by design - "upload button" is not a valid ARIA role. See [ARIA spec](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html) for details. We're correctly implementing the ARIA spec by setting the role `menuitem` on the command bar, which is a navigational menu.
![image](https://user-images.githubusercontent.com/4161791/76569109-5b629400-6480-11ea-9ad0-8b79b19e05ff.png)
